### PR TITLE
Fix linter errors

### PR DESCRIPTION
**Pull request by roborourke**
This fixes the current HM Linter errors for the module.

### CODE REVIEW

**Comment by roborourke** [comment]
@rmccue this one is good to go now.

File: inc/about/namespace.php [hunk 1]
```diff
@@ -204,7 +209,7 @@ function render_credits_page() {
 			<p>Altis includes other libraries as part of WordPress.</p>
 			<?php
 			array_walk( $libraries['data'], '_wp_credits_build_object_link' );
-			echo '<p class="wp-credits-list">' . wp_sprintf( '%l.', $libraries['data'] ) . "</p>\n\n";
+			echo '<p class="wp-credits-list">' . sprintf( '%s.', wp_kses( implode( ', ', $libraries['data'] ), [ 'a' => [ 'href' => [] ] ] ) ) . "</p>\n\n";
```
Comment (rmccue) [inline inc/about/namespace.php:212]: Could use `whitelist_html` here instead now that's in Altis? Also, no need for `sprintf( '%s'`, can move the `.` back into the string at the end.

Comment (roborourke) [reply inc/about/namespace.php:212]: Oh yeah duh, the problem is our coding standards aren't aware of `whitelist_html()` as an escaping function.

Comment (rmccue) [reply inc/about/namespace.php]: Ah! Added that in HM-Minimum I think

**Pull request merged**